Tribler/Core/defaults.py

```diff
@@ -251,6 +252,7 @@
 tribler_defaults['Tribler']['window_height'] = 670
 tribler_defaults['Tribler']['sash_position'] = -185
 tribler_defaults['Tribler']['family_filter'] = 1
+tribler_defaults['Tribler']['monochrome_tray_icon'] = False
```

Comment:
Please don't add this setting to the core, it's a GUI setting.

Take a look at how we toggle the debug mode in the GUI; you should do the same with this setting.